frontend/javascripts/oxalis/model/sagas/save_saga.ts

```diff
@@ -988,33 +990,54 @@ export function performDiffTracing(
 
   return actions;
 }
+
+let isWkReady = false;
+
+function setWkReady() {
+  isWkReady = true;
+}
```

Comment:
adding to my previous comment: how about moving the wk-ready related code into a separate module and hooking it directly up in the root saga (no dependence on the save_saga). then, there could be a separate saga like this:

```
export function* ensureWkReady() {
  if (isWkReady) return;
  yield* take("WK_READY");
}
```

then, `ensureWkReady` would be an easy way to ensure that wk is ready. might make sense to use it for other sagas, too, but that's obviously not necessary for this PR.

Reply:
That's a great idea :+1: